apps/passport-client/components/screens/FrogScreens/ScoreTab.tsx

```diff
@@ -1,15 +1,67 @@
-// renders under construction in a retro style ASCII art of frog
-export function ScoreTab() {
+import { requestFrogCryptoGetScores } from "@pcd/passport-interface";
+import { FrogCryptoScore } from "@pcd/passport-interface/src/FrogCrypto";
+import { useEffect, useState } from "react";
+import styled from "styled-components";
+import { appConfig } from "../../../src/appConfig";
+import { RippleLoader } from "../../core/RippleLoader";
+
+export function ScoreTab({ score }: { score?: FrogCryptoScore }) {
+  const [scores, setScores] = useState<FrogCryptoScore[]>([]);
+  useEffect(() => {
+    requestFrogCryptoGetScores(appConfig.zupassServer).then((res) => {
+      setScores(res.value || []);
+    });
+  }, []);
```

Comment:
a little weird that `score` is passed in but `scores` are loaded from within

Reply:
True :) Only because we get user score from the user state and we also display user score in GetScore tab.